src/ui/include/docengine.h

```diff
@@ -74,7 +75,7 @@ class DocEngine : public QObject
     bool read(QFile *file, Editor *editor);
     bool read(QFile *file, Editor *editor, QTextCodec *codec, bool bom);
     // FIXME Separate from reload
-    bool loadDocuments(const QList<QUrl> &fileNames, EditorTabWidget *tabWidget, const bool reload, QTextCodec *codec, bool bom);
+    bool loadDocuments(const QList<QUrl> &fileNames, EditorTabWidget *tabWidget, const bool reload, QTextCodec *codec, bool bom, bool rememberLastSelectedDir=true);
```

Comment:
I was thinking about giving loadDocument a once-over with another Pull Request. I left it out of this one because it'd just make things messier. So hopefully the `rememberLastSelectedDir` won't actually stay there for long.